src/controller/OpenRobertaController.vala

```diff
@@ -75,6 +75,7 @@ namespace BrickManager {
                     });
                 }, () => {
                     status_bar_item.connected = false;
+                    open_roberta_window.connected = false;
```

Comment:
Not sure how I missed the ticket. The docs for python-dbus are in a pittyful state, so I hope I changing this is somewhat straightforward.
I'll definitely add a null-check in the code above. Having this change in is still better than not having it for now though. Not sure what the timelines for releasing all this is.